kubernetes/ClusterSubscription/resource.yaml

```diff
@@ -69,5 +69,11 @@ spec:
                 configMapKeyRef:
                   name: razee-identity
                   key: CLUSTER_ID
+            - name: RAZEE_UPDATE_INTERVAL_MS
+                valueFrom:
+                  configMapKeyRef:
+                    name: razee-identity
```

Comment:
We may want to have a separate `clustersubscription-overrides` configmap because `razee-identity` is probably not overly accessible.

Reply:
Updated 👍